config/secrets.yml

```diff
@@ -130,9 +130,9 @@ test:
   should_send_notifications: true
   # Sendgrid
   # a real api_key is needed for recording new VCR based tests
-  sendgrid_api_offline: true
+  sendgrid_api_offline: false
   sendgrid_api_key: <%= ENV["SENDGRID_API_KEY"] || "fakeapikey" %>
-  sendgrid_publishers_list_id: 3648346
+  sendgrid_publishers_list_id: <%= ENV["SENDGRID_PUBLISHERS_LIST_ID"] || 3648346 %>
```

Comment:
Given the `sendgrid_publishers_list_id` is in the urls and/or body of the request we'll get matching errors if this isn't consistent. 

If we're having issues it's probably because the cassettes were recorded against two different accounts. We should upgrade all cassettes and this value to use the id from the new shared dev sendgrid account.

Reply:
`3986776` is the list id used by the shared acct